_benchmark/user-guide/concepts.md

```diff
@@ -20,95 +24,61 @@ Before using OpenSearch Benchmark, familiarize yourself with the following conce
 
 - **Test**: A single invocation of the OpenSearch Benchmark binary.
 
-A workload is a specification of one or more benchmarking scenarios. A workload typically includes the following:
+## Test concepts
 
-- One or more data streams that are ingested into indexes.
-- A set of queries and operations that are invoked as part of the benchmark.
+At the end of each test, OpenSearch Benchmark produces a table that summarizes the following: 
 
-## Throughput and latency
+- [Concepts](#concepts)
+  - [Core concepts and definitions](#core-concepts-and-definitions)
+  - [Test concepts](#test-concepts)
+    - [Differences between OpenSearch Benchmark and a traditional client-server system](#differences-between-opensearch-benchmark-and-a-traditional-client-server-system)
+  - [Processing time](#processing-time)
```

Comment:
Can we ident this since processing time needs to be in the same bullet pattern as the rest of the test concepts?